runtime/port/unix/j9sysinfo.c

```diff
@@ -257,6 +257,19 @@ static intptr_t getAIXPPCDescription(struct J9PortLibrary *portLibrary, J9Proces
 #define __power_10() (_system_configuration.implementation == POWER_10)
 #endif /* !defined(__power_10) */
 
+#if !defined(__power_11)
+#define POWER_11 0x80000 /* Power 11 class CPU */
+#define __power_11() (_system_configuration.implementation == POWER_11)
+#endif /* !defined(__power_11) */
```

Comment:
Rather than have processor recognition in both OMR and OpenJ9, I suggest this be removed in favour of using the support from OMR.

Reply:
I will suggest opening a separate PR to address the issue of this existing technical debt (might have others as well), instead of blocking this P11 short-term solution.

Reply:
Adding more dependencies on the duplicate code in OpenJ9 is going in the wrong direction.

Reply:
The code that you anticipate that would use the new constant (`PROCESSOR_PPC_P11`) should use the available detection code in OMR instead. This only enables more technical debt.

Reply:
notice the existing separate constants to begin with (`PROCESSOR_PPC_P11` doesn't exist in OMR).  should-use is not applicable at this point. we all knew what is the ultimate approach we desire for, but that is out of scope for this PR as I mentioned. At least, before we invest enough resources and understand the exact situation (OMR/OpenJ9 use what features respectively and how we can unify them), we should not start to use partial code from either. Just for better sanity's sake.

Reply:
No. I still don't see the point of further duplication between OMR and OpenJ9.

Reply:
although we knew something bigger and better should be done (in the long run), there is little reason to hold this PR hostage to the ultimate goal of code form. why is it not a reasonable approach to open a separate issue to address the long-term technical debt?

Reply:
If you share where you anticipate checking for POWER11, I would be happy to help show how the functionality in OMR can be used there.

Reply:
is it difficult to look up where CPU testing is used?

Reply:
if you have a reasonable design to unify them or eliminate openj9 side of things in mind already, i would strongly recommend opening a separate issue to do it in the near-mid term.  in the mean time, merge this one to fulfill this project short-term: https://github.ibm.com/runtimes/semeru-releases/issues/95

Reply:
We're long overdue for removal of this duplication.

Searching the code for `PROCESSOR_PPC_P*`, I all references are already the names from OMR (`OMR_PROCESSOR_PPC_P6`, `OMR_PROCESSOR_PPC_P7`, `OMR_PROCESSOR_PPC_P8`, etc.). The only occurrences of `PROCESSOR_PPC_P*` (not from OMR) are those in `j9port.h` and `j9sysinfo.c` It appears those could simply be removed now. I will explore that, in which case we have no need for this at all.

Reply:
sounds good.  go ahead and do it then.

Reply:
I'm inclined to close this. The places that I can imagine where POWER11 needs to be recognized appear already to be using the OMR functionality, so `OMR_PROCESSOR_PPC_P11` should be used instead of `PROCESSOR_PPC_P11` anyway.

Reply:
Please don't only imagine. you need to make sure of it. technically, merely close this without the appropriate compensation, POWER11 will be known to OpenJ9 side as unknown ppc processor as it is (you just need to walk down the current code to know this). Whether there are implications of it ... i haven't investigated it thoroughly.

Reply:
You haven't offered any clues where `PROCESSOR_PPC_P11` **_might_** be used; what else am I to do?

Reply:
that is a question you should answer to begin with (why CPU-recognition code existed in OpenJ9 side?).

Reply:
Although as I mentioned in my comment, j9sysinfo_get_processor_description() isn't used on P. I'm guessing it's there for completeness and because nobody wanted to remove it before.

Reply:
It may be something else in j9sysinfo that is specific to Java, not necessarily j9sysinfo_get_processor_description(). I don't know if we thought about removing parts of the j9 implementation or just kept it as is when OMR was created.